pkg/types/aws/platform.go

```diff
@@ -59,6 +62,20 @@ type Platform struct {
 	// AWS resources that the operators create.
 	// +optional
 	PropagateUserTag bool `json:"propagateUserTags,omitempty"`
+
+	// LBType allows user to set a load balancer type.
+	// When this field is set the default ingresscontroller will get created using the specified LBType.
+	// If this field is not set then the default ingress controller of LBType Classic will be created.
+	// Valid values are:
+	// * "Classic": A Classic Load Balancer that makes routing decisions at either
+	//   the transport layer (TCP/SSL) or the application layer (HTTP/HTTPS). See
+	//   the following for additional details:
+	//     https://docs.aws.amazon.com/AmazonECS/latest/developerguide/load-balancer-types.html#clb
+	// * "NLB": A Network Load Balancer that makes routing decisions at the
+	//   transport layer (TCP/SSL). See the following for additional details:
+	//     https://docs.aws.amazon.com/AmazonECS/latest/developerguide/load-balancer-types.html#nlb
+	// +optional
+	LBType configv1.AWSLBType `json:"lbType,omitempty"`
```

Comment:
Nit: this comment is adding weird spacing in the generated docs.

Reply:
@r4f4  does it look OK to you now ?